### ISSUE TITLE
Surface Accessibility Reports Artifacts via Email

### ISSUE THREAD

**Issue by asksep** [microsoft/accessibility-insights-action#1269]
## Describe the bug
Filing issue on behalf of user based on feedback:
"It would be awesome to be able to surface the accessibility report artifacts via email, like Azure pipeline notifications, so we don't have to keep digging for them."

### Problem
Accessibility Insights ADO Extension does not have a way to surface reports via email.

### Ask
Create a method to surface deduplicated Accessibility Reports Artifacts via email.

### Examples: 
Azure Pipeline Notifications

**Comment by katydecorah** [comment]
There are a couple ways to improve the discoverability of the artifacts report:


## Improve error message logging

If the action finds errors, it will display a node process exit error message which does not have meaningful context to the user:

> Exit code 1 returned from process: file name '/home/vsts/agents/2.206.1/externals/node16/bin/node', arguments '"/home/vsts/work/_tasks/accessibility-insights_dc031a10-21c6-5672-9e59-a8d92a9ba7f4/4.0.0/index.js"'. 


### Screenshots

Email notification error:

![screenshot of Azure DevOps email notification with the exit code error message](https://user-images.githubusercontent.com/2180540/187754908-80a2d16a-b9fb-40e9-a88e-aead8e4ba58e.png)

Pipeline error:

![screenshot of Azure pipeline where the exit code error message displays](https://user-images.githubusercontent.com/2180540/187756017-ae53a212-e3d4-452c-b5a3-f9319054e023.png)


### Suggested fix

We can use the azure-pipe-lines-task-lib [setResult](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/docs/azure-pipelines-task-lib.md#tasksetResult%27) to create a custom message, for example: 

> Scan completed with errors, visit the build's Extensions tab for full details and to download the accessibility report.


This will reduce ambiguity over the error and provide the user with next actions.



## Update link text in the summary upload

When you open the extension tab, the link to the artifact's report may not be easily discoverable as it uses "run artifacts" as the link text:

> See scan details by downloading the report from [run artifacts](#)

### Screenshots

Azure pipeline Extensions tab:

![screenshot of extensions tab](https://user-images.githubusercontent.com/2180540/187755181-9f8c0e06-3b3b-44fa-9cb8-3f40f55b3a6e.png)


### Suggested fix

To improve discoverable, we can update the link text to something more descriptive:

> [Download the accessibility report](#)
